edi_pdf2data/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Pdf2data Import",
+    "summary": """
+        Module that allows to import data from a pdf""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Creu Blanca,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/edi",
+    "depends": ["edi"],
+    "external_dependencies": {"python": ["invoice2data", "PyMuPDF"]},
```

Comment:
We should not depend on these libs here IMO.
Here we should have the core machinery to be able to define parsers for PDF data.

Reply:
You are right, now is the moment to make it new. Let me check it :smile:

Should we reuse the idea from invoice2data or should we make it new?

Reply:
IMO invoice2data cen be used. 
Currently we are developing some interesmsting featuresfor next release:
Improved lines parser
Improved tesseract support (auto detect language)
Pdfplumber input support

Currently working on implementing Excalibur/Camelot for advanced table parsing. It is a lot of work, probably not finished for next release.


However Pymupdf is not yet supported as a input parser.


The biggest drawback of the module is to write/store the templates.

Reply:
Well, the name of the library is confusing too... It is called invoice2data, but could be used for other kind of documents. Something like pdf2data would have been better IMO.

We have already used it to import docs and generate specific records.

Reply:
I think the name of the library has been discussed before, but kept unchanged.
It can always be used to parse pdf's with other content.
Currently I am working on integrating camelot and invoice2data. Very handy tool for importing table data from pdf's.

Reply:
Also, the problem is that it is focused on invoices, so you need to disallow some fields if you want to import other kind of data. With the new changes, it is easier for the user to make it, as it is done entirely on web interface. Not using yaml anymore (but you can import invoice2data yml :smile: )